### DISCUSSION TITLE
Cross-app automation triggering

### DISCUSSION THREAD

**Discussion by jaimes-alten** [Budibase/budibase#13480] [Help] [answered]
Hello,

Say you have 2 apps (A and B), and app A contains automation A1. Is it possible to trigger automation A1 from app B? Currently it seems you can only trigger an automation from another automation within the same app.

Should I submit this as a feature request?

Thanks

**Comment by andz-bb** [comment, marked as answer]
hey, you should be able to set up an automation in app A that is [triggered by a webhook](https://docs.budibase.com/docs/webhook), and then post to that webhook using a [REST query in App B's data section.](https://docs.budibase.com/docs/rest-queries) Hope this helps - if you run into any trouble with this let me know!

**Reply by jaimes-alten** [reply]
Thanks @andz-bb good idea!
